src/app/tabs/newAdmin/domainInfo/components/RenewButtonComponent.js

```diff
@@ -18,9 +18,11 @@ const RenewButtonComponent = (props) => {
     <p>
       {disable ? strings.domain_expired : `${strings.expires_on} ${formatDate(dayMath(expires))}`}
 
-      <Button onClick={handleClick} className={isRenewOpen ? 'active caps-first' : 'caps-first'} disabled={disable}>
-        {strings.renew}
-      </Button>
+      {disable ? '' : (
```

Comment:
small point, I leave to you preference:
!disable && 
would also work and is a bit cleaner IMO, since the first option is not used